src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -117,14 +117,26 @@ gpr_timespec TimeoutSecondsToDeadline(int64_t seconds) {
 
 }  // namespace
 
+static const int64_t MINIMUM_FILE_WATCHER_REFRESH_INTERVAL_SECONDS = 1;
```

Comment:
Is this how constants are defined elsewhere? I think people usually use `kVar` for constants. Capital letters with underscore are usually macros.

Reply:
I also feel like `static` is not necessary here. `constexpr` usually suffices.

Reply:
I'm not sure exactly what the style is, I'll dig around some more code

I saw these - https://source.corp.google.com/github/grpc/grpc/src/core/tsi/ssl_transport_security.cc;rcl=1d8fac36b3914fc17296158d4cb90b3fe51f4654;l=68 but those _are_ macros so I suppose I shouldn't be re-using that naming pattern

Reply:
Changed to `static constexpr kVar`